next-cloudinary/src/lib/cloudinary.js

```diff
@@ -104,6 +104,28 @@ export function getPublicId(src) {
 
   return src;
 }
+export function getTransformations(src, useUrlTransformations) {
```

Comment:
can you add a little comment header for the function and maybe a little @description along with it?